cmake/CMakeLists.txt

```diff
@@ -1214,6 +1217,20 @@ if (onnxruntime_USE_TVM)
   list(APPEND onnxruntime_EXTERNAL_DEPENDENCIES tvm nnvm_compiler)
 endif()
 
+if (onnxruntime_ENABLE_EXTENSION_CUSTOM_OPS)
+  # add compile definition to enable custom operators in onnxruntime-extensions
+  add_compile_definitions(ENABLE_EXTENSION_CUSTOM_OPS)
+
+  # set options for onnxruntime-extensions
+  set(OCOS_ENABLE_CTEST OFF CACHE INTERNAL "")
+  set(OCOS_ENABLE_STATIC_LIB ON CACHE INTERNAL "")
+  set(OCOS_ENABLE_SPM_TOKENIZER OFF CACHE INTERNAL "")
```

Comment:
Why is sentencePieceTokenizer disabled?

Reply:
Currently **SentencepieceTokenizer** is disabled because it will use **protobuf-lite** lib which has conflicts with the **protobuf** used in ONNXRuntime, and when we build it will throw errors: error LNK2005: *** function already defined in libprotobuf-lite.lib, fatal error LNK1169: one or more multiply defined symbols found.
I plan to fix it in the future because it may need to change the code in **sentencepiece** project to allow using the protobuf lib from ONNXRuntime. Do you have any different ideas on this? Thanks.

Reply:
I see, thanks